src/js/components/Video/stories/typescript/TheaterMode.tsx

```diff
@@ -25,9 +25,9 @@ const TheaterMode = (props) => {
               a11yTitle: 'Toggle theater mode',
             },
             'fullScreen',
+            'pause',
```

Comment:
```suggestion
```
I don't think we need to include the pause button in this example. Or if we do include it we should also have the play button.

Reply:
aw yes no this was me testing TS